src/main/java/common/blocks/Block_LapotronicEnergyUnit.java

```diff
@@ -4,21 +4,51 @@
 
 import common.itemBlocks.IB_LapotronicEnergyUnit;
 import cpw.mods.fml.common.registry.GameRegistry;
+import gregtech.api.enums.Textures;
+import gregtech.api.interfaces.IIconContainer;
+import gregtech.api.render.TextureFactory;
+import gregtech.api.util.GT_Utility;
 import net.minecraft.block.Block;
 import net.minecraft.block.material.Material;
 import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.client.renderer.texture.TextureMap;
 import net.minecraft.creativetab.CreativeTabs;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.IIcon;
+import net.minecraft.util.ResourceLocation;
 
 public class Block_LapotronicEnergyUnit extends BaseGTUpdateableBlock {
-	
+
 	private static final Block_LapotronicEnergyUnit INSTANCE = new Block_LapotronicEnergyUnit();
-	
+    public enum IconBaseSide implements IIconContainer {
+        INSTANCE;
+        IconBaseSide() {
+            // technically, this page is owned by me, glee8e, however, I'm kind enough to spare 1 for kekztech since
```

Comment:
Greg style humor tradition?

Reply:
More or less to not make an unsuspecting third person to allocate in page 42 again in kekztech and eventually run into conflict with my own stuff.